src/uikit.cs

```diff
@@ -13652,35 +13654,40 @@ public partial interface UIPopoverPresentationController {
 	[Protocol, Model]
 	[BaseType (typeof (NSObject))]
 	public partial interface UIAdaptivePresentationControllerDelegate {
+		[IgnoredInDelegate]
 		[Export ("adaptivePresentationStyleForPresentationController:")]
 		UIModalPresentationStyle GetAdaptivePresentationStyle (UIPresentationController forPresentationController);
 	
-		[Export ("presentationController:viewControllerForAdaptivePresentationStyle:")]
+		[Export ("presentationController:viewControllerForAdaptivePresentationStyle:"), 
+			DelegateName ("AdaptivePresentationRequestedWithStyle"), DefaultValue (null)]
```

Comment:
The WithStyle has to be left 'cause we have a WithTraits too. But I'm adding the UI prefix to follow the Apple style (even when we both angry that is not very .Net ;) )